docs/how-to/input-output/gcs_duckdb.qmd

```diff
@@ -0,0 +1,23 @@
+---
+title: Loading Google Cloud Storage files with DuckDB
+---
+
+Loading data stored on Google Cloud Storage to DuckDB requires specifying `gcs` as filesystem.
+
+```{python}
```

Comment:
````suggestion
```python
````

When you wrap `python` in braces like this, quarto will execute the code block. Here we **don't** want to execute because you're not actually specifying a URL.